src/library_wasmfs_opfs.js

```diff
@@ -169,9 +169,17 @@ mergeInto(LibraryManager.library, {
   },
 
   _wasmfs_opfs_insert_file__deps: ['$wasmfsOPFSGetOrCreateFile'],
-  _wasmfs_opfs_insert_file: async function(ctx, parent, namePtr, childIDPtr) {
+  _wasmfs_opfs_insert_file: async function(ctx, parent, namePtr, childIDPtr, errPtr) {
```

Comment:
Maybe we can simplify this by returning a negative number via the `childIDPtr` rather than adding a new argument?

Reply:
Hmm, perhaps, but that seems slightly risky. We might use more than 2GB ids in theory some day, like if we use pointers in linear memory somehow, and in wasm64.

I lean towards keeping this, especially as it's consistent with the other changes you are making - I copied it from there?

Reply:
Although actually this is different from what we do in `open_access`. I guess we would ideally fix `open` to use this pattern as well.